WordPress/src/main/java/org/wordpress/android/models/Post.java

```diff
@@ -395,12 +395,12 @@ public String getQuickPostType() {
      * the user first publishes a post
      * @return
      */
-    public boolean hasChangedFromLocalDraftToPublished() {
-        return mChangedFromLocalDraftToPublished;
+    public boolean hasChangedFromDraftToPublished() {
```

Comment:
I renamed it because the post changed its status from Draft to Published. 
"LocalDraft" is normally used for device only posts.